README.md

```diff
@@ -104,9 +104,9 @@ let watcher = watcher(api, ListParams::default());
 This now gives a continual stream of events and you do not need to care about the watch having to restart, or connections dropping.
```

Comment:
Not really your fault, but this is pretty misleading when the example code below fails early on a watch error (using `?`). Ideally it should just log the error and move on.

Reply:
Yeah, good point. We don't technically have to care about the watcher errors, but we can't bubble them up with `?` either, which puts the onus of handling error  cases - that looks handled from watcher internals - on the user.

Maybe we should have an interface that doesn't bubble up errors? Possibly a bit in dragon-territory because we don't want to end up error-spamming internally if misconfigured, but it might be worth doing. What do you think?

Reply:
I have to admit that I didn't even read the text :( I just tried to make the code compile.
I'm away from the computer most of the week and can't improve this at the moment. Feel free to change my PR though!

Reply:
I'd say that's definitely in dragon territory, since then we also need to provide things like a backoff policy, and more or less make the library unusable without the application using the same logging strategy as the library..

Reply:
One thing we /could/ do would be to add a `Stream` helper for these kinds of things..

Reply:
Yeah, I wasn't thinking this would be mandatory, this feels like helper territory. I'll make a comment on #296 to continue this there.

Thanks @lfrancke ! Yeah, at a certain point I gloss over our text and markdown a bit myself (hence this type of state).